tests/serial/time/explicit/serial-coupling/ReadWriteScalarDataWithWaveformSubcyclingFirst.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<precice-configuration>
+  <solver-interface dimensions="3">
+    <data:scalar name="DataOne">
+      <waveform order="1" />
+    </data:scalar>
```

Comment:
Why do you use a waveform if the testcase is called "subcycling". Is this again a naming problem?

Reply:
Generally You can use a waveform, if you are subcycling. The waveform interpolates linearly between the beginning and the end of the window. You subcycle within this window and sample from the waveform. I do not see a real problem here. Maybe something we need to discuss?

Reply:
We can also try to discuss naming issues in the documentation PR: https://github.com/precice/precice.github.io/pull/109

If some definitions of terms are missing, we should definitely add them somewhere in the documentation and then use these terms consistently here.

Reply:
I see. For me "subcycling" is the old constant-extrapolation-way of doing things, but maybe I need to forget this somehow wrong knowledge once and for all :see_no_evil: